<ui version="4.0" >
 <class>GuiTransferTemplates</class>
 <widget class="QWidget" name="GuiTransferTemplates" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Transfer Templates</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="spacing" >
    <number>6</number>
   </property>
   <property name="margin" >
    <number>9</number>
   </property>
   <item>
    <widget class="QLabel" name="textLabel1" >
     <property name="text" >
      <string>&lt;h1>Transfer Templates&lt;/h1></string>
     </property>
     <property name="wordWrap" >
      <bool>false</bool>
     </property>
     <property name="margin" >
      <number>2</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget" >
     <property name="selectionMode" >
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text" >
       <string>Recipient</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Value</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Purpose</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Bank Id</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Account Id</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBoxButtons" >
     <property name="focusPolicy" >
      <enum>Qt::TabFocus</enum>
     </property>
     <property name="title" >
      <string>Use Template for a</string>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>6</number>
      </property>
      <property name="margin" >
       <number>9</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButtonSingleTransfer" >
        <property name="text" >
         <string>Single Transfer</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDatedTransfer" >
        <property name="text" >
         <string>Dated Transfer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonStandingOrder" >
        <property name="text" >
         <string>Standing Order</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonDebitNote" >
        <property name="text" >
         <string>Debitnote</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="spacing" >
      <number>6</number>
     </property>
     <property name="margin" >
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="buttonCancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
       <property name="shortcut" >
        <string>Alt+C</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>1</width>
         <height>21</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="buttonClearPurpose" >
       <property name="text" >
        <string>Clear Purpose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonDelete" >
       <property name="text" >
        <string>&amp;Delete Template</string>
       </property>
       <property name="shortcut" >
        <string>Alt+D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonUse" >
       <property name="text" >
        <string>&amp;Use Template</string>
       </property>
       <property name="shortcut" >
        <string>Alt+U</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
